dascore/io/core.py

```
@@ -392,72 +411,112 @@ def _eps(self):
-        for form in formats:
-            entries = [name for name in self._eps.index if name.startswith(form)]
-            for name, loader in self._eps.loc[entries].items():
-                fiberio = self._load_entry_point(name, loader)
-                if fiberio is not None:
-                    self.register_fiberio(fiberio)
-            if form not in self._format_version:
-                self._failed_formats.add(form)
-        # The selected format(s) should now be loaded
-        assert set(formats).isdisjoint(self.unloaded_formats)
-        return
+        format = format.upper() if format is not None else None
```

Ruff (0.15.21)

[error] Variable `format` is shadowing a Python builtin

(A001)